l10n_it_website_sale_fiscalcode/README.rst

```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+Website Sale FiscalCode
+=======================
+
+This module adds: 
+* Fiscalcode on checkout form
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Go to Shop.
+* Buy something.
+* On checkout form and add fiscalcode
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/113/9.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
```

Comment:
Please use the latest version of `Bug Tracker` section according to https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst